portal/oauth/README.metadata.json

```diff
@@ -1,34 +1,26 @@
 {
     "category": "Portal",
-    "description": "Authenticate with OAuth 2.0 to retrieve the user's profile information.",
+    "description": "Use OAuth2 to authenticate with ArcGIS Online (or your own portal) to access secured resources.",
```

Comment:
These need to be pretty short for the sample viewer. How about just "Use OAuth2 to authenticate with a secure portal."?